.msync.yml

```diff
@@ -1 +1 @@
-modulesync_config_version: '2.10.1'
+modulesync_config_version: '2.11.0'
```

Comment:
Is this because 2.12.0 is a pre-release?

Reply:
yes